.travis.yml

```diff
@@ -43,6 +43,26 @@ matrix:
     - python: "3.6"
       env: TOXENV=py36-flake8
 
+    # Dashboard JS
+    - language: node_js
+      node_js: "8"
+      cache: npm
+      before_install:
+        - export DISPLAY=:99.0
+        - sh -e /etc/init.d/xvfb start
+        - sleep 3 # give xvfb some time to start
+        - cd ./src/dashboard/frontend/
```

Comment:
Wow @replaceafill that was fast. Why do we still need xvfb?

Reply:
We don't. And this made me realize we don't need the browser drivers either in the `dashboard.Dockerfile`. I just removed them in https://github.com/artefactual/archivematica/pull/1364/commits/36bba916ca5930caa9f2aad13a070048f67dd988